### PR TITLE
SensorService: don't contact batterystats service

### PR DESCRIPTION
**Pull request by peat-psuwit**
This service is normally in Java part. minimediaservice have a stub
implementation for this service, but some port may choose not to run it.

Change-Id: I20aa2f00d6f55ac14a77ad33bde10b51e26b9967

### CODE REVIEW

**Comment by luksus42** [comment]
I am not sure, if this breaks another functionality, but it fixes the wake lag issue on my FP2.

File: services/sensorservice/BatteryService.cpp [hunk 1]
```diff
@@ -91,13 +91,15 @@ void BatteryService::cleanupImpl(uid_t uid) {
 }
 
 bool BatteryService::checkService() {
+#if 0
```
Comment (Flohack74) [inline services/sensorservice/BatteryService.cpp:94]: If you disable code unconditionally then I see no point in keeping it around, I would vote for a simple delete. Dead code in either comments or disabled like this just bloats the project.

**Review by Flohack74 — CHANGES_REQUESTED** [review]

File: services/sensorservice/BatteryService.cpp (hunk 1 above)
Comment (peat-psuwit) [reply services/sensorservice/BatteryService.cpp:94]: I leave it in to reduce potential conflicts when merging code from upstream, should the upstream modifies the code around here.

Comment (Flohack74) [reply services/sensorservice/BatteryService.cpp:94]: Ok thats clever actually...

**Review by Flohack74 — APPROVED** [review]

**Pull request merged**